packages/backend/config/settings.py

```diff
@@ -313,6 +313,14 @@
 STRIPE_LIVE_MODE = env.bool("STRIPE_LIVE_MODE", default=False)
 DJSTRIPE_WEBHOOK_SECRET = env("DJSTRIPE_WEBHOOK_SECRET", default="")
 DJSTRIPE_FOREIGN_KEY_TO_FIELD = "id"
+
+
+def tenant_request_callback(request):
+    from apps.multitenancy.models import Tenant # Import models here to avoid an →``AppRegistryNotReady`` exception
```

Comment:
This is breaking backend tests. Why do you need this?

Reply:
@wojcikmat can you please take a look at this? We added this change to be able to list all the cards.

Reply:
@mskwierczynski could you try to remove line 319, it should help? This import is not needed and linter doesn't like it 😄 so in the body of the function we should have only:
```
def tenant_request_callback(request):
    return request.tenant
```